FusionIIIT/applications/online_cms/models.py

```diff
@@ -1,3 +1,137 @@
+                self.pk, self.course_id,
+                self.start_time, self.end_time,
+                self.negative_marks)
+
+
+class QuizQuestion(models.Model):
+    quiz_id = models.ForeignKey(Quiz, on_delete=models.CASCADE)
+    question = models.TextField(max_length=1000)
+    options1 = models.CharField(null=True, max_length=100)
+    options2 = models.CharField(null=True, max_length=100)
+    options3 = models.CharField(null=True, max_length=100)
+    options4 = models.CharField(null=True, max_length=100)
+    options5 = models.CharField(null=True, max_length=100)
+    answer = models.IntegerField()
+    announcement = models.TextField(max_length=2000, null=True, blank=True)
+    image = models.TextField(max_length=1000, null=True)
+    marks = models.IntegerField()
+
+    def __str__(self):
+        return '{} - {} - {} - {} - {} - {} - {} - {} - {}'.format(
+            self.pk, self.quiz_id, self.options1,
+            self.options2, self.options3, self.options4,
+            self.options5, self.answer, self.announcement)
+
+
+class StudentAnswer(models.Model):
+    student_id = models.ForeignKey(Student, on_delete=models.CASCADE)
+    quiz_id = models.ForeignKey(Quiz, on_delete=models.CASCADE)
+    question_id = models.ForeignKey(QuizQuestion, on_delete=models.CASCADE)
+    choice = models.IntegerField()
+
+    def __str__(self):
+        return '{} - {} - {} - {} - {}'.format(
+                self.pk, self.student_id,
+                self.quiz_id, self.question_id,
+                self.choice)
+
+
+class Assignment(models.Model):
+    course_id = models.ForeignKey(Course, on_delete=models.CASCADE)
+    upload_time = models.DateTimeField(auto_now=True)
+    submit_date = models.DateTimeField()
+    assignment_name = models.CharField(max_length=100)
+    assignment_url = models.CharField(max_length=100, null=True)
+
+    def __str__(self):
+        return '{} - {} - {}'.format(self.pk, self.course_id, self.assignment_name)
+
+
+class StudentAssignment(models.Model):
+    student_id = models.ForeignKey(Student, on_delete=models.CASCADE)
+    assignment_id = models.ForeignKey(Assignment, on_delete=models.CASCADE)
+    upload_time = models.DateTimeField(auto_now=True)
+    upload_url = models.TextField(max_length=200)
+    score = models.IntegerField()
+    feedback = models.CharField(max_length=100)
+
+    def __str__(self):
+        return '{} - {} - {} - {} - {}'.format(
+                self.pk, self.student_id,
+                self.assignment_id, self.score,
+                self.feedback)
+
+
+class QuizResult(models.Model):
+    quiz_id = models.ForeignKey(Quiz, on_delete=models.CASCADE)
+    student_id = models.ForeignKey(Student, on_delete=models.CASCADE)
+    score = models.IntegerField()
+    feedback = models.CharField(max_length=100)
+
+    def __str__(self):
+        return '{} - {} - {} - {} - {}'.format(
+                self.pk, self.student_id,
+                self.qid, self.score,
```

Comment:
Got one here. qid should be replaced by quiz_id

Reply:
done